### PR TITLE
Kubernetes policy and install docs/demo

### PR DESCRIPTION
**Pull request by gunjan5**

### CODE REVIEW

File: master/getting-started/kubernetes/tutorials/stars-policy/index.md [hunk 1]
```diff
@@ -87,3 +87,5 @@ kubectl create -f policies/frontend-policy.yaml
 
 The client can now access the frontend, but not the backend.  Neither the frontend nor the backend
 can initiate connections to the client.  The frontend can still access the backend.
+
+For advanced network policy demonstration, check out the [advanced network policy demo]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/tutorials/advanced-policy).
```
Comment (caseydavenport) [inline master/getting-started/kubernetes/tutorials/stars-policy/index.md]: I'd make this more specific - "To use Calico to enforce egress policy on Kubernetes pods, see the advanced policy demo"

**Comment by caseydavenport** [comment]
A few comments, then LGTM.

File: master/getting-started/kubernetes/tutorials/advanced-policy.md [hunk 2]
```diff
@@ -138,7 +139,7 @@ We'll define some network policy through the Kubernetes API.  Run the following
 a NetworkPolicy which allows traffic to nginx pods from any pods in the advanced-policy-demo Namespace. 
 
 ```shell
-kubectl create -f - <<EOF 
+cat << EOF | kubectl create -f -
```
Comment (caseydavenport) [inline master/getting-started/kubernetes/tutorials/advanced-policy.md]: I prefer the other way :)

It doesn't require `cat`

Comment (gunjan5) [inline master/getting-started/kubernetes/tutorials/advanced-policy.md]: yeah I also like it better, didn't know we could do it that way. I think we should also change all the calicoctl ones to use this (in a separate PR) to be consistent. I can do that

Comment (caseydavenport) [inline master/getting-started/kubernetes/tutorials/advanced-policy.md]: Sound fine to me.  Changing the calicoctl ones isn't release critical though, so please hold off until after v2.0 :)

**Comment by caseydavenport** [comment]
Ship it!

**Pull request merged**